src/parser.js

```diff
@@ -927,6 +927,11 @@ class Parser {
                         : !!plural;
                 })();
 
+                let contextValues = [];
+                if (containsContext) {
+                    contextValues = options.context === '' ? ['male', 'female'] : [options.context];
```

Comment:
May I know why it needs to specify a hard-coded value `['male', 'female']` in the parser?

Reply:
Ha, you right, `context` can be anything?

I've dug a little bit in the code, and saw this test https://github.com/i18next/i18next/blob/270904f6369ee9bbda059c3186fcea7baf9eb15d/test/translator/translator.translate.context.spec.js#L49, 
That means that `context` can't be anything

Reply:
Ok, so I've checked, and looks like `context` can be any thing.
What do you thing will be a good option?